src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang

```diff
@@ -0,0 +1,44 @@
+# GUI
+itemGroup.pedestalcrafting=Pedestal Crafting
+
+config.displaySpeed=Affichage de la vitesse
+config.displaySpeed.tooltip=Vitesse à laquel les éléments tournent, s'élèvent et tombent sur les piédestal - rotations/sec
+
+gui.jei.category.pedestal=Piédestal
+
+# Items
+tile.pedestal.name=Piédestal
+tile.pedestal.diamond.name=Piédestal en diamant
+tile.pedestal.emerald.name=Piédestal en émeraude
+tile.pedestal.glowstone.name=Piédestal en pierre lumineuse
+tile.pedestal.gold.name=Piédestal en or
+tile.pedestal.grass.name=Piédestal en herbe
+tile.pedestal.iron.name=Piédestal en fer
+tile.pedestal.lapis.name=Piédestal en lapis
+tile.pedestal.logacacia.name=Piédestal en bois d'accacias
```

Comment:
"accacias" devrait être "acacia"

Reply:
You're right, I corrected it in my last commit